integration/spec_helper.rb

```diff
@@ -0,0 +1,52 @@
+# encoding: utf-8
+
+require "bundler/setup"
+require "backup"
+
+Dir[File.expand_path("../support/**/*.rb", __FILE__)].each { |f| require f }
+
+require "rspec/autorun"
```

Comment:
If you rebase again master I've updated RSpec and this require won't work anymore